pandas/core/arrays/datetimes.py

```diff
@@ -1141,7 +1142,7 @@ def to_pydatetime(self) -> npt.NDArray[np.object_]:
         """
         return ints_to_pydatetime(self.asi8, tz=self.tz, reso=self._creso)
 
-    def normalize(self) -> Self:
+    def normalize(self, *args, **kwargs) -> Self:
```

Comment:
There is no `normalize` numpy function as far as I know so this should not be included